presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorSplitManager.java

```diff
@@ -34,6 +39,16 @@ default ConnectorSplitSource getSplits(
         throw new UnsupportedOperationException();
     }
 
+    default ConnectorSplitSource getSplits(
+            ConnectorTransactionHandle transaction,
+            ConnectorSession session,
+            ConnectorTableHandle table,
+            SplitSchedulingStrategy splitSchedulingStrategy,
+            List<Future<DynamicFilterDescription>> dynamicFilters)
+    {
+        return getSplits(transaction, session, table, splitSchedulingStrategy);
```

Comment:
presto-spi pom currently does not include io.airlift.log-manager dependency, I'm not sure whether it's ok to add it.
@sopel39 WDYT ?

Reply:
for logging io.airlift.log dependency should be enough I guess. What if we add them in the `SplitManager` ?

Reply:
@Praveen2112 I'm not sure if we can achieve what we want in SplitManager. Unless I'm missing something, we'll know that `getSplits` with dynamic filters has not been implemented either in ConnectorSplitManager in spi or in each individual plugin's SplitManager.

Reply:
@raunaqmorarka In that case what if we use `java.util.logging.Logger` and it will be tunneled through their logging subsystem. `io.airlift.log.Logger` is just a wrap over `java.util.logging.Logger`

Reply:
Made changes to log using `java.util.logging.Logger` here